deploy/ansible/hosts

```diff
@@ -1,8 +1,9 @@
-[centos7]
-35.187.116.62
+[hosts]
+104.155.94.58
```

Comment:
Looks strange for me, maybe just replace with localhost and provide doc with "how to write this files"?

Reply:
I will not work with localhost in `hosts` file, ansible handles local connection in a different way